### ISSUE TITLE
leak of logits

### ISSUE THREAD

**Issue by TerryWu0414** [wangzrk/LMHaze#4]
Hello, while reproducing your work, I noticed that your preprocessed logit evaluation script only includes five tokens. However, in the MoE network structure, I saw that 14 expert modules are utilized. Could you please clarify which additional tokens were used as prior information in the logit evaluation?
